application/LicenseAppService.go

```diff
@@ -162,26 +169,48 @@ func (s *LicenseAppService) ProcessOrgEntitledEvent(evt OrgEntitledEvent) error
 	if err != nil {
 		return err
 	}
-	//skip userImport when a new license is added but the org/members already exist.
-	if orgIsImported {
-		glog.Infof("License for service %s with %v seats added. Skipping user import. Org already exists.", evt.ServiceID, evt.MaxSeats)
-		return nil
+
+	// always run import.
+	_, e := s.importUsers(evt.OrgID)
+	if e != nil {
+		return err
 	}
+	return nil
```

Comment:
we could easily change this now to also return the counts btw, but this PR is already big enough. If there's a wish, I'm happy to add it ;)